examples/NAT.cs

```diff
@@ -193,28 +193,28 @@ sealed class InternalNode : IEquatable<InternalNode>
   {
     private readonly IPAddress _Address; // FIXME we should really use immutable values since we are using a dictionary
     private readonly ushort _Port;
-    private readonly int _NetworkPort;
+    private readonly int _InterfaceNumber;
```

Comment:
I haven't added a comment, but I changed the name. Do you think it is clear enough what Port is?

> I can intuit what's intended to be stored in _Port and how it differs from _NetworkPort (partly from the types of both variables), but owing to the similarity of their names it would be useful to have a comment that states what each of the two is intended to store.